angularjs/Dialog/appearance-and-styling.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Syncfusion Dialog appearance-and-styling
-description: appearance and styling
+title: Appearance and styling in Angularjs Dialog control | Syncfusion
```

Comment:
Change `AngularJS`, check and change in all places.

Reply:
changed